mod-source-record-manager-server/src/main/java/org/folio/services/QuickMarcEventProducerServiceImpl.java

```diff
@@ -52,16 +52,16 @@ private Future<Boolean> sendEventInternal(String eventPayload, String eventType,
     try {
       var event = createEvent(eventPayload, eventType, tenantId);
       var topicName = createTopicName(eventType, tenantId, kafkaConfig);
-      var record = createProducerRecord(event, key, topicName, kafkaHeaders);
+      var producerRecord = createProducerRecord(event, key, topicName, kafkaHeaders);
       var producer = kafkaProducers.get(eventType);
       if (producer != null) {
-        producer.write(record)
-          .onSuccess(unused -> {
+        producer.send(producerRecord)
+          .onSuccess(res -> {
             LOGGER.info("sendEventInternal:: Event with type {} was sent to kafka", eventType);
             promise.complete(true);
           })
-          .onFailure(throwable -> {
-            var cause = throwable.getCause();
+          .onFailure(err -> {
```

Comment:
Could you please explain, what is the point of this refactoring?

Reply:
perhaps, I shouldn't have touched it, since it just tautology, I roll it back to previous version